### PR TITLE
Maintenance update

### LINKED ISSUE #111: Cleanup unwanted files
We have found unwanted files commited to this repository:
 - [ ] .idea
 - [ ] .idea/.gitignore
 - [ ] .idea/modules.xml
 - [ ] .idea/nest-prisma-template.iml
 - [ ] public/readme
 - [ ] public/readme/profile.png
 - [ ] public/readme/secrets.png
 - [ ] public/readme/user.png

These files, present many issues as various build manifests can contain secrets (generated during build time) or unwanted images/files from template repositories can be copied to container images, which slows build time, etc.
We also decided not to support `.idea` settings, if they are present.

Please, remove above mentioned files and make necessary adjustments to `.gitignore` and `.dockerignore` so these will not be commited and used again.
Also, check if there aren't any more files that we might have missed that are unnecessary or unwanted and can be removed.

### LINKED ISSUE #113: Upgrade Dockerfile
We have found old version of a these Dockerfiles:
 - [ ] [Dockerfile](https://github.com/bratislava/nest-clamav-scanner/blob/master/Dockerfile)

The old version has many security issues and bad practicies! Please, update to newer version as soon as possible. 
You can find refrences to newer versions here: 
 - [Next Dockerfile](https://github.com/bratislava/next-strapi-template/blob/master/next/Dockerfile)
 - [Strapi Dockerfile](https://github.com/bratislava/next-strapi-template/blob/master/strapi/Dockerfile)
 - [Nest Dockerfile](https://github.com/bratislava/nest-prisma-template/blob/master/Dockerfile)

### PR DESCRIPTION
**Pull request by skifahrer**
- updated dependencies
- updated prisma to v5 and fixed related errors
- updated node version
- added volta settings and removed .nvmrc
- updated Dockerfile
- removed unwanted files

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,57 +1,43 @@
...
+COPY --chown=node:node --from=build /build/prisma ./prisma
 COPY --chown=node:node nest-cli.json ./nest-cli.json
-
-ENTRYPOINT npx prisma migrate deploy && npm run start:prod
\ No newline at end of file
+EXPOSE 3000
+ARG GIT_COMMIT="undefined"
+ENV GIT_COMMIT=$GIT_COMMIT
+LABEL org.opencontainers.image.revision="${GIT_COMMIT}" \
+      org.opencontainers.image.source="https://github.com/bratislava/nest-prisma-template" \
```
Comment (vidriduch) [inline Dockerfile:41]:
```suggestion
      org.opencontainers.image.source="https://github.com/bratislava/nest-clamav-scanner/" \
```

File: Dockerfile [hunk 2]
```diff
@@ -1,57 +1,43 @@
...
+      org.opencontainers.image.licenses="EUPL-1.2"
+CMD [ "npm", "run", "start:migrate:prod" ]
```
Comment (vidriduch) [inline Dockerfile:43]: I might be overlooking something but does this command `start:migrate:prod` exists?

**Review by vidriduch — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (skifahrer) [reply Dockerfile]: thanks, i did overlooked it

File: Dockerfile (hunk 2 above)
Comment (skifahrer) [reply Dockerfile:43]: thanks, i did overlooked it, now fixed

**Review by vidriduch — APPROVED** [review]

**Pull request merged**